Jellyfin.Server.Implementations/Item/PeopleRepository.cs

```diff
@@ -42,25 +42,23 @@ public IReadOnlyList<PersonInfo> GetPeople(InternalPeopleQuery filter)
         // Include PeopleBaseItemMap
         if (!filter.ItemId.IsEmpty())
         {
-            var people = dbQuery.ToArray();
-            var peopleIds = people.Select(p => p.Id).ToArray();
-
-            var mappings = context.PeopleBaseItemMap
-                .AsNoTracking()
-                .Where(m => peopleIds.Contains(m.PeopleId) && m.ItemId == filter.ItemId)
-                .ToDictionary(m => m.PeopleId);
-
-            return people.Select(p =>
-            {
-                var personInfo = Map(p);
-                if (mappings.TryGetValue(p.Id, out var mapping))
+            var query = dbQuery
```

Comment:
Would you mind please also adding a comment here or if you are willing implement the todo?

```suggestion
            // TODO: This is not the ideal solution to the problem of people entries not loading. It would be best to add the relation to the entity and use the .Include EFCore method to load the referenced role and sorting order instead
            var query = dbQuery
```

Reply:
Sure, I can change it to use Include instead. Probably won't be able to be at the computer for a week though, so it'll have to wait a lil bit probably.

Reply:
I've changed it to .Include now instead